src/client/extension.ts

```diff
@@ -299,6 +333,23 @@ export async function deactivate(): Promise<void> {
     disposeNotificationPanel();
 }
 
```

Comment:
This also we can move to a separate file along with above code.

Reply:
done